pontoon/static/js/tags/admin/button.js

```diff
@@ -0,0 +1,51 @@
+
+import React from 'react';
+
+import {TagResourceManager} from './manager';
+
+
+export default class TagResourcesButton extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {open: false};
+    }
+
+    handleClick = (evt) => {
+        evt.preventDefault();
+        this.setState((prevState) => ({open: !prevState.open}));
+    }
+
+    renderButton () {
+        const {open} = this.state;
+        let message = "Manage resources for this tag";
+        if (open) {
+            message = "Hide the resource manager for this tag";
+        }
+        return (
+              <button onClick={this.handleClick}>
+                {message}
+              </button>);
+    }
+
+    renderResourceManager () {
+        const {open} = this.state;
+        if (!open) {
+            return '';
+        }
+        return (
+            <TagResourceManager
+               tag={this.props.tag}
+               api={this.props.api}
+               requestMethod="post"
+               project={this.props.project} />);
+    }
+
+    render () {
+        return (
+            <div>
+              {this.renderButton()}
+              {this.renderResourceManager()}
+            </div>);
```

Comment:
I don't understand the indentation you use in your `render` functions. I see some 6-space, 2-space and even 3-space in this file alone. Wouldn't it be much simpler if you used 4-space everywhere?

Reply:
im using `rjsx-mode` in emacs which defaults to 4 space on code, 2 on jsx - which i like tbh

Reply:
Fair enough, but why use 3-space for attributes in JSX?

Reply:
well - again - i was going with the defaults for the mode, which i quite like